drivers/wifi/nxp/nxp_wifi_drv.c

```diff
@@ -322,6 +322,12 @@ int nxp_wifi_wlan_event_callback(enum wlan_event_reason reason, void *data)
 		net_if_dormant_on(g_uap.netif);
 		LOG_DBG("WLAN: UAP Stopped");
 
+		if (net_addr_pton(AF_INET, CONFIG_NXP_WIFI_SOFTAP_IP_ADDRESS, &dhcps_addr4) < 0) {
+			LOG_ERR("Invalid CONFIG_NXP_WIFI_SOFTAP_IP_ADDRESS");
```

Comment:
shouldn't this check and error be done at the START? What's the point of doing this at STOP?

Reply:
start alreay has the check, if get the address failed, it will return.